python-shell/src/generate.py

```diff
@@ -0,0 +1,12 @@
+import os
+
+from gafferpy.gaffer_connector import GafferConnector
+from fishbowl.fishbowl import Fishbowl
+
+# Generate the core api using spring-rest as it has access to every store operation
+gc = GafferConnector("http://localhost:8080/rest")
```

Comment:
What's the convention you're using to decide whether ' or " are being used?

Reply:
I presume you are asking `http://localhost:8080/rest` or `http://localhost:8080/rest/latest`?
`http://localhost:8080/rest` is used here because it is used for the `spring-rest` endpoint, which is what is needed to generate the full api.

Reply:
In this instance I'm actually meaning in the general case what determines whether an apostrophe or a quote is used for a string literal. Every theory I've come up with seems to be incorrect at some point.

Reply:
Haha yes another good point! This needs to be made consistent across all gafferpy, probably should be tackled in: https://github.com/gchq/gaffer-tools/issues/137. Right now it's probably dependent on who wrote the code